### PR TITLE
Update docs for softmax in onnx supported operators

### PR DESCRIPTION
**Pull request by hackeritchy**
Update the softmax in onnx supported operators from `softmax (only dim=-1 supported)` to `softmax`, as all cases of dim options are supported in:
[#18482](https://github.com/pytorch/pytorch/pull/18482): ONNX Export All Cases of Softmax

### CODE REVIEW

**Review by bddppq — APPROVED** [review]
Thanks
